Library_WebServer/Database/LibraryDbContext.cs

```diff
@@ -6,7 +6,7 @@ namespace Library_WebServer.Database
 {
     public class LibraryDbContext : DbContext
     {
-        private const string CONNECTION_STRING = "Server=localhost;Port=5432;Database=library;User Id=pgadmin;Password=pgpass;";
+        private const string CONNECTION_STRING = "Server=localhost;Port=5432;Database=library;User Id=admin;Password=admin;";
```

Comment:
I don't mind it in this case, and we discussed it, but just as a reminder it's good to double check for changes like that and remove them before a PR.